wagtail/admin/tests/test_page_chooser.py

```diff
@@ -202,6 +202,51 @@ def test_with_invalid_page_type(self):
         response = self.get({'page_type': 'foo'})
         self.assertEqual(response.status_code, 404)
 
+    def test_with_admin_display_title(self):
+        # Check the display of the child page title when it's a child
+        response = self.get({'page_type': 'wagtailcore.Page'})
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'wagtailadmin/chooser/browse.html')
+
+        self.assertInHTML(self.child_page.get_admin_display_title(), response.json().get('html'))
```

Comment:
Rather than comparing against `self.child_page.get_admin_display_title()` here, it's better to be explicit about the result we're expecting, i.e.:

    self.assertInHTML("foobarbaz (simple page)", response.json().get('html'))

This way, it's easier to confirm that the test is doing what it set out to do, just from looking at the test itself - as it stands, I find myself asking "is this test passing because the bug is fixed, or because `get_admin_display_title` just happens to return the plain title instead of the custom one?" (For example, we might be mistakenly testing on a page model that hasn't overridden `get_admin_display_title`, or `self.child_page` might be the non-specific Page object at this point in the code - it's necessary to cross-reference with other bits of the test code to confirm that isn't the case.) Please can you update these assertions (here and in the other tests) so that we're comparing against an explicit result?

Reply:
Thanks, I've made that change now. I chose to leave the reference to `child_page.id` in the breadcrumb match because it' not really what we're testing here - Happy to change that too if you don't think that's the right choice

Reply:
Thanks! Yep, leaving `child_page.id` in place is fine - there's no real way around it, as you can't guarantee that the database will reliably assign it the same ID each run.